core/serv/src/main.rs

```diff
@@ -262,7 +253,10 @@ enum CliCommand {
 impl CliCommand {
     pub async fn run_command(self, ctx: &CliCtx) -> Result<CommandOutput> {
         match self {
-            CliCommand::Commands(command) => command.run_command(ctx).await,
+            CliCommand::Commands(command) => {
+                set_logging("warn", &Path::new(""), &vec![])?;
```

Comment:
ok, let it be. Just  with `info` as a default log level

Reply:
I used warn, because on info we get unwanted output. For example:
```
$ yagna id list
[2021-01-07T08:56:33Z INFO  ya_service_bus::remote_router] trying to connect to: 127.0.0.1:7464
[2021-01-07T08:56:33Z INFO  ya_service_bus::connection] started connection to gsb
┌───────────┬──────────┬─────────┬──────────────────────────────────────────────┐
│  default  │  locked  │  alias  │  address                                     │
├───────────┼──────────┼─────────┼──────────────────────────────────────────────┤
│  X        │          │         │  0x0a50cb00895929cf40a4c2e323989dbaa72c801b  │
└───────────┴──────────┴─────────┴──────────────────────────────────────────────┘
[2021-01-07T08:56:33Z INFO  ya_service_bus::connection] stopped connection to gsb
```